pyproject.toml

```diff
@@ -36,7 +35,7 @@ pytest = "^7.2.2"
 [tool.poetry.extras]
 django = ["django"]
 flask = ["flask"]
-fastapi = ["fastapi"]
+asgi = []
```

Comment:
With no dependencies, installing with the "asgi" extra doesn't actually do anything, right? Makes me wonder if it's worth having the extra, but I guess it keeps our installation process consistent (users are always using an extra depending on their framework). Is that what you're thinking?

Reply:
That's exactly right. I was considering leaving this out, but it feels weird to install everything else as an extra, but not ASGI support...

I was going for consistency here. Functionally it doesn't do anything.

Reply:
I'm on board with that!